building/build-debs/homeworld-admin-tools/src/verify.py

```diff
@@ -44,6 +44,16 @@ def check_keystatics():
     print("pass: keyserver serving correct static files")
 
 
+def check_ssh(node, command):
+    config = configuration.Config.load_from_project()
```

Comment:
A session object wouldn't be worthwhile, because this isn't a long-running process. It's probably worth separating out a helper like `configuration.get_config()`, which could cache the configuration -- but I don't think it matters much, because it's not like loading the configuration is a bottleneck.

Reply:
It's not so much performance as the fact that you're effectively using the config file as a global variable. This is not consistent with, for example, setup.py where functions take in the configuration as a parameter -- if I were to call such a function with a modified configuration, I would expect it to behave according to that configuration. If you're intent on configuration being global, it would be more explicit to stick it in a global variable and reference that everywhere the configuration is used.

By passing in a session object, I meant something like wrapop in setup.py. In fact, Operations could itself serve as the session object, with some changes.

Alternatively, the configuration could just be passed in as an additional parameter to check_ssh.

Reply:
The reason that the functions have that in setup.py is just for convenience -- all of those methods were originally intended to *only* be called automatically. If that's confusing, the correct solution is to eliminate those parameters, and just letting everyone use get_config().

Configuration being global is intentional, because it avoids passing around unnecessary state and simplifies the code.

I'll go and eliminate some of the places where it's stored or passed around, to make this more clear.